cmd/gateway/vald/main.go

```diff
@@ -40,12 +40,12 @@ func main() {
 			context.Background(),
 			runner.WithName(name),
 			runner.WithVersion(info.Version, maxVersion, minVersion),
-			runner.WithConfigLoader(func(path string) (interface{}, string, string, error) {
+			runner.WithConfigLoader(func(path string) (interface{}, config.Common, error) {
```

Comment:
If not available config.Common should be nil so please change this type to pointer

Reply:
than you for your review.
I fixed